### ISSUE TITLE
Org image size not updated for transcluded text

### ISSUE THREAD

**Issue by joshcho** [nobiot/org-transclusion#127]
When I change the image size in org (using image-decrease-size), the change doesn't get reflected in the transcluded text.

Calling the function on the transcluded text does update the size, but once you remove and add again, the image is back to its default size.

**Comment by nobiot** [comment]
I have a little bit of hard time to really reproduce the issue, but I think I got what you are doing -- I don't use `org-toggle-inline-images` often and every time I refresh the transclusion with an image I need to disable and then enable it again. 

For keeping the decreased image size, I don't know if i can do much. 

But I set this variable `org-image-actual-width` to make the default a reasonable size for my screen. I suggest that you use it rather than adjusting the size manually.

**Comment by joshcho** [comment]
Yup, that's what I ended up doing, and it works well until I have different kinds of images (textbook screenshots and slides require very different image sizes). I will try to look into this when I get the time.

**Comment by nobiot** [comment]
How about this solution?
https://stackoverflow.com/questions/11670654/how-to-resize-images-in-org-mode/69339844#69339844

You should be able to add #+attr* before specific transclusion keywords

**Comment by ParetoOptimalDev** [comment]
> How about this solution? https://stackoverflow.com/questions/11670654/how-to-resize-images-in-org-mode/69339844#69339844
> 
> You should be able to add #+attr* before specific transclusion keywords

I can confirm this works as I just used it.
